rrfs-test/validated_yamls/templates/obtype_config/abi_g18.yaml

```diff
@@ -124,12 +124,18 @@
 
        # Data Thinning
        - filter: Gaussian Thinning
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g18_channels
          #horizontal_mesh: 145
          horizontal_mesh: 60
          use_reduced_horizontal_grid: true
-         distance_norm: geodesic
-       #  round_horizontal_bin_count_to_nearest: true
-       #  partition_longitude_bins_using_mesh: true
+         distance_norm: maximum
+         time_mesh: 'PT06H'
+         time_min: '2024-05-26T22:30:00Z'
+         time_max: '2024-05-27T01:30:00Z'
```

Comment:
Should time_mesh be PT03H since a 3-hour window is used here?

Reply:
Sorry for the delayed response. You are right, I have changed the PT06H to PT03H in the new push.